ch02.markdown

```diff
@@ -2469,44 +2469,44 @@ Pour plus d'information, voir l'article [Mixins dans
 Dart.](/articles/mixins/)
 
 
-### Class variables and methods {#class-variables-and-methods}
+### Variables de classes et méthodes {#class-variables-and-methods}
 {:.no_toc}
 
-Use the `static` keyword to implement class-wide variables and methods.
+Utilisez le mot clé `static` pour implémenter des variables et méthodes de classes.
 
-#### Static variables {#static-variables}
+#### Variables statiques {#static-variables}
 {:.no_toc}
 
-Static variables (class variables) are useful for class-wide state and
-constants:
+Les variables statiques (variables de classes) font utiles pour class-wide state et
```

Comment:
sont utiles pour
class-wide state and constants => un état et des constantes communs à la classe ?